src/selectors/user.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
```

Comment:
:nit 2020 (?)

Reply:
will do in a commit in the other PR before merging into the feature branch